src/marker.rs

```diff
@@ -665,12 +670,23 @@ impl MarkerExpression {
     /// use std::collections::HashSet;
     /// use std::str::FromStr;
     /// use pep508_rs::MarkerTree;
+    /// use pep440_rs::Version;
```

Comment:
NIT: You can hide the `use` statements from readers so that they can focus on the logic (or other logic that is not relevant for the example)

```suggestion
    /// # use std::collections::HashSet;
    /// # use std::str::FromStr;
    /// # use pep508_rs::MarkerTree;
    /// # use pep440_rs::Version;
```